argostime/templates/stats.html.jinja

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html.jinja" %}
+{% block title %}Statistieken | Argostimè{% endblock %}
+{% block content %}
+<h1>Statistieken</h1>
+<ul>
+<li>Totaal aantal producten: {{ total_products }}</li>
+<li>Totaal aantal aanbiedingen van producten: {{ total_offers }}.</li>
```

Comment:
Ik denk dat voor veel gebruikers niet heel duidelijk zal zijn wat hier precies staat. Het is denk ik niet heel duidelijk wat het verschil tussen de 1e en de 2e is. Het woord "aanbiedingen" heeft ook een dubbele betekenis.